### PR TITLE
[0.52-$] compilation fix.

### PR DESCRIPTION
**Pull request by Lucarda**
This PR includes 'draw-shapes-help.pd' on autotools so that compilation finishes.

The file was renamed.

### CODE REVIEW

**Comment by umlaeute** [comment]
merged into `develop`. thanks a lot.

**Comment by porres** [comment]
sorry :/ I first made doc commits to the wrong master branch and when I moved them to the proper branch I missed this :/

File: doc/Makefile.am [hunk 1]
```diff
@@ -53,450 +53,457 @@ empty=
...
-     ./8.topics/slop-slew-limit.png \
-     ./8.topics/slop-slew-limiting-patch.png \
-     ./8.topics/slop-tilde-1-curves.png \
-     ./8.topics/slop-tilde.htm \
-     ./Makefile.am \
-     ./sound/bell.aiff \
-     ./sound/voice.wav \
-     ./sound/voice2.wav \
+     ./1.manual/1.introduction.txt
+     ./1.manual/favicon.ico
```
Comment (umlaeute) [inline doc/Makefile.am:57]: these lines all miss the line continuation backslash.

**Review by umlaeute — CHANGES_REQUESTED** [review]
hmm.
right before the changes there is a 2-line instruction on how to properly generate the list of files
(including the trailing slash).

did this not work for you?

File: doc/Makefile.am (hunk 1 above)
Comment (Lucarda) [reply doc/Makefile.am]: I correct it on 9197e15

**Comment by Lucarda** [comment]
> right before the changes there is a 2-line instruction on how to properly generate the list of files
(including the trailing slash).

&#128584;

closing this :)

**Comment by umlaeute** [comment]
actually there were some more missing files (`Pd-messages.pd` and `ds-test.pd`) that i'm just going to push to `devel`

**Comment by Lucarda** [comment]
Yeah. That why I was adding commits. No problem as you are doing `develop`. &#128077;

**Pull request closed without merging**